spec/controllers/application_controller/dialog_runner_spec.rb

```diff
@@ -2,6 +2,39 @@
 require "support/controller_spec_helper"
 
 describe CatalogController do
+  describe "#dialog_field_changed" do
+    include_context "valid session"
+
+    let(:dialog) { active_record_instance_double("Dialog") }
+    let(:wf) { double(:dialog => dialog) }
+    let(:dialog_field) do
+      active_record_instance_double("DialogFieldTextBox", :name => "test", :value => nil, :type => "TextBox")
+    end
+    let(:params) { {:test => "new value", :id => 123} }
+    let(:session) { {:edit => {:rec_id => 123, :wf => wf, :key => "dialog_edit__123"}} }
+
+    before do
+      Dialog.stub(:find_by_id).with(123).and_return(dialog)
+      dialog.stub(:field)
+      dialog.stub(:field).with("test").and_return(dialog_field)
+      dialog.stub(:field_name_exist?).and_return(false)
+      dialog.stub(:field_name_exist?).with("test").and_return(true)
+      dialog.stub(:dialog_tabs).and_return([double(:dialog_groups => [double(:dialog_fields => [dialog_field])])])
+      wf.stub(:dialog_field).with("test").and_return(double(:data_type => "string"))
+      wf.stub(:set_value)
+    end
+
+    it "includes disabling the sparkle in the response" do
```

Comment:
@himdel I would imagine we can change it to only do `miqSparkle(false)` somewhere in the javascript now, I think when this was created we didn't have as much control over ajax transactions as we do now. In this case since this PR was about drop-downs we would just call `miqSparkle(false)` in the `miqSelectPickerEvent` callback function.